docs/qubo_tools/pathfinder/JSON.md

```
@@ -0,0 +1,37 @@
+    "settings": {
+        "encoding": one of ["ONE_HOT", "UNARY", "DOMAIN_WALL", "BINARY"],
+        "n_paths": integer,
+        "max_path_length": integer,
+        "loops": boolean,
+    },
+    "objective_function": Constraint,
+    "constraints": array[Constraint]
+}
+```
+
+Individual constraints are defined based on their JSON definitions provided [here](https://github.com/munich-quantum-toolkit/problemsolver/blob/main/src/mqt/problemsolver/qubo_tools/pathfinder/resources/constraints).
```

markdownlint-cli2 (0.18.1)

Link text should be descriptive

(MD059, descriptive-link-text)